docs-new/docs/contribute/general/refinement-guide.md

```diff
@@ -0,0 +1,49 @@
+# Refinement timeblock
+
+During Refinement timeblock, maintainers engage in technical
+discussions on open issues and pull requests.
+This dedicated time allows for in-depth conversations, knowledge sharing,
+and collective decision-making.
+It is an opportunity for the team to synchronize their understanding of
+ongoing developments, address challenges, and ensure a common vision for the project.
+
+**Purpose and goals:**
+
+- **Alignment:** Ensure a shared understanding among maintainers regarding ongoing
+developments, project goals, and upcoming tickets.
+- **Technical Depth:** Delve into the technical aspects of open issues and pull
+requests, facilitating a deeper understanding of proposed Pull requests.
```

Comment:
are you sure you're talking about technical **depth** and not technical **debt**? 🤔

Reply:
I would say it's about technical depth rather than technical debt since its focus is on understanding the technical details part more